src/search/product.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * The structure of a product.
```

Comment:
Does this comment provide any value?

Reply:
Very minimally. It feels incomplete, both in the code and in the generated docs, to leave it bare. We're open to suggestions on improving the wording.

Reply:
All I can think of is elaborating on the meaning of 'product', but I'm not sure what to say about it.